openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1289,6 +1289,8 @@ Cache-Control: no-store
 }
 ```
 
+In the event the Credential Issuer can no longer issue the credential(s), the `credential_request_denied` error code as defined in (#credential-request-errors) should be used in response to a request. A wallet upon receiving this error SHOULD stop making requests to the deferred credential endpoint for the given `transaction_id`. 
```

Comment:
Probably will is slightly better than can. Maybe we should go to 'MUST' instead of 'SHOULD'? I can live with either I guess.

```suggestion
In the event the Credential Issuer will no longer issue the credential(s), the `credential_request_denied` error code as defined in (#credential-request-errors) should be used in response to a request. A wallet upon receiving this error SHOULD stop making requests to the deferred credential endpoint for the given `transaction_id`. 
```

Reply:
I read this sentence as guidance to credential issuers, hence `can` is a bit more directed to them as the subject of the sentence. Not that concerned about adopting the change if others feel strongly.